scenarios/import_td_logs/config/workflow_log.yml

```diff
@@ -0,0 +1,40 @@
+in:
+  type: sftp
+  host: ${dla_host}
+  user: ${user_id}
+  secret_key_file: {"content": "${secret:sftp.dla_secret_key_file}"}
+  path_prefix: "/treasure-data-logs/production/${account_id}/workflow_logs/v1/data.csv"
```

Comment:
account id is possibly not unique across regions. Better to use account_key such a 'aws_1' Or 'production' can be production_us, production_tokyo, production_eu01, production_ap02

Reply:
Understand. Will fix.